code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -195,11 +195,56 @@
 	item_state = "bomber"
 	icon_open = "bomber_open"
 	icon_closed = "bomber"
+	style = STYLE_LOW
 	body_parts_covered = UPPER_TORSO|ARMS
 	cold_protection = UPPER_TORSO|ARMS
 	min_cold_protection_temperature = T0C - 20
 	siemens_coefficient = 0.7
 
+/obj/item/clothing/suit/storage/toggle/bomber/furred
```

Comment:
Since all these jackets share so much stats, refactor them under 
/obj/item/clothing/suit/storage/toggle/jacket ?

It would make adding more than much faster